cmd/kflex/common/cp.go

```diff
@@ -18,10 +18,45 @@ package common
 
 import (
 	"context"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
FYI, almost everywhere this gets named "metav1".

Reply:
fixed, thank you